cli/tools/repl/session.rs

```diff
@@ -379,6 +395,38 @@ impl ReplSession {
     })
   }
 
+  async fn check_for_npm_imports(
+    &mut self,
+    program: Arc<swc_ast::Program>,
+  ) -> Result<(), AnyError> {
+    let mut collector = ImportCollector::new();
+    program.visit_with(&mut collector);
+
+    let npm_imports = collector
+      .imports
+      .iter()
+      .filter(|i| i.starts_with("npm:"))
+      .flat_map(|i| ModuleSpecifier::parse(i))
+      .collect::<Vec<ModuleSpecifier>>();
+    if !npm_imports.is_empty() {
+      self.proc_state.prepare_node_std_graph().await?;
+      crate::node::initialize_runtime(&mut self.worker.js_runtime).await?;
```

Comment:
Does this get executed with every npm import?

Reply:
Probably not, I'll check that